src/main/org/nlogo/app/AgentMonitorManager.scala

```diff
@@ -153,6 +153,17 @@ with org.nlogo.window.Event.LinkParent
         new Runnable() { def run() { frame.requestFocus() }})
   }
 
+  def stopInspecting(agent: Agent) {
+    if(agent != null) {
```

Comment:
I actually had that thought when I wrote it, but I was just mirroring the code above